pkg/ui/workspaces/cluster-ui/src/transactionDetails/recentTransactionDetails.tsx

```diff
@@ -114,38 +114,34 @@ export const RecentTransactionDetails: React.FC<
           </Col>
         </Row>
         {transaction && (
-          <Row gutter={24}>
+          <Row gutter={24} type="flex">
             <Col className="gutter-row" span={12}>
               <SummaryCard className={cx("summary-card")}>
-                <Row>
-                  <Col>
-                    <SummaryCardItem
-                      label={RecentTxnInsightsLabels.START_TIME}
-                      value={transaction.start.format(
-                        "MMM D, YYYY [at] H:mm (UTC)",
-                      )}
-                    />
-                    <SummaryCardItem
-                      label={RecentTxnInsightsLabels.ELAPSED_TIME}
-                      value={Duration(
-                        transaction.elapsedTime.asMilliseconds() * 1e6,
-                      )}
-                    />
-                    <SummaryCardItem
-                      label={RecentTxnInsightsLabels.STATUS}
-                      value={
-                        <>
-                          <StatusIcon status={transaction.status} />
-                          {transaction.status}
-                        </>
-                      }
-                    />
-                    <SummaryCardItem
-                      label={RecentTxnInsightsLabels.PRIORITY}
-                      value={capitalize(transaction.priority)}
-                    />
-                  </Col>
-                </Row>
+                <SummaryCardItem
+                  label={RecentTxnInsightsLabels.START_TIME}
+                  value={transaction.start.format(
+                    "MMM D, YYYY [at] H:mm (UTC)",
```

Comment:
can you update this to use `DATE_FORMAT_24_UTC`

Reply:
Yup, is there an issue tracking this too? I can include it in the PR

Reply:
No issue, I just noticed on this PR while reviewing

Reply:
Done.